### PR TITLE
Di 24289

### PR DESCRIPTION
**Pull request by saargt**

### CODE REVIEW

File: shared/src/usersCore.service.ts [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+import { SchemaManipulator } from "./schemaManipulator";
+import { Request } from "@pepperi-addons/debug-server";
+import IPapiService from "./IPapi.service";
+
 
 export class UsersCoreService extends BaseCoreService
 {
+	constructor(schema: AddonDataScheme, request: Request, papiService: IPapiService)
+	{
+		super(schema, request, papiService);
```
Comment (ShirMarciano) [inline shared/src/usersCore.service.ts:13]: This constructor is duplication of the AccountUsersCoreService -  It is not good

**Review by ShirMarciano — CHANGES_REQUESTED** [review]
please no code duplication, If we will need to add modification date or creation date we will need to do it in two places instead of one

File: shared/src/schemaManipulator.ts [hunk 2]
```diff
@@ -13,8 +13,26 @@ export class SchemaManipulator
      * @param fieldName {string} The name of the field to add
      * @param field {SchemeField} The field to add
      */
-	public addField(fieldName: string, field: SchemeField): void
+	protected addField(fieldName: string, field: SchemeField): void
 	{
         this.schema.Fields![fieldName] = field;
```
Comment (ShirMarciano) [inline shared/src/schemaManipulator.ts:18]: We have lint on this addon? 
I

Comment (saargt) [reply shared/src/schemaManipulator.ts:18]: We do. Not the standard Pepperi one, but it has what I used to work with.
Same as core resources.

**Pull request merged**